src/lib/Seasons.js

```diff
@@ -624,7 +624,7 @@ const eastertide = y => {
         week: Math.floor( i / 7 ) + 1
       }),
       data: {
-        prioritized: true,
+        //prioritized: true, // TODO: check rule in easter time with ferials and (opt) memories
```

Comment:
```js
         //prioritized: true, // TODO: Check rule in Eastertide with ferias and (optional) memories
```

Reply:
This todo was just a note for me, but I'll remove this line.
The `prioritized: true` was making all the feria prioritized in Eastertide, meaning that all optional memories was dropped. But in the General Instruction of the Roman Missal (point 355), it's mentioned that it's still allowed to celebrate them during this season.